avroparquet/src/main/scala/akka/stream/alpakka/avroparquet/impl/AvroParquetFlow.scala

```diff
@@ -0,0 +1,45 @@
+package akka.stream.alpakka.avroparquet.impl
+import akka.stream.{Attributes, FlowShape, Inlet, Outlet}
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+import org.apache.avro.generic.GenericRecord
+import org.apache.parquet.hadoop.ParquetWriter
+
+private[avroparquet] class AvroParquetFlow(writer: ParquetWriter[GenericRecord])
```

Comment:
Add `@InternalApi` and a ScalaDoc comment stating `INTERNAL API` to all internal classes.

Reply:
Done.